src/AElf.OS.Network.Grpc/GrpcServerService.cs

```diff
@@ -132,8 +132,8 @@ private GrpcPeer DialPeer(string peerAddress, Handshake handshake)
                 StartHeight = handshake.Header.Height,
                 IsInbound = true
             };
-
-            return new GrpcPeer(channel, client, connectionInfo);
+            
```

Comment:
do not pass EventBus

Reply:
So can we put the peer in the DI system (create with GetService<IPeer>()) ? this way the logger will be injected.
I can also see another solution, we can throw an exception here (maybe PeerDisconnectedException) and move the logic to the NetworkService. This way GrpcPeer does not need the EventBus.

Reply:
Moved the logic for handling failing peers from GrpcPeer to the NetworkService.